asciidoctorj-core/src/main/java/org/asciidoctor/jruby/ast/impl/ContentNodeImpl.java

```diff
@@ -34,6 +34,8 @@ public String getContext() {
         return getString("context");
     }
 
+    @Override
+    public void setContext(String context) { setString("context", context); }
```

Comment:
Could you please format this in the same way as the rest of the file?
We usually have line breaks after curly braces.

Reply:
Yep, can do that.